assets/scss/views/_about.scss

```diff
@@ -36,14 +36,31 @@ main {
               @include flex-row;
 
               .member {
-                @include flex-row;
-
-                flex-basis: 50%;
-                flex-wrap: nowrap;
-                padding: 0 ($whitespace * 1.5) ($whitespace) 0;
+                border-radius: $radius;
+                display: flex;
+                flex-direction: row;
+                flex:  calc(50% - #{$whitespace});
+                margin: 0 $whitespace $whitespace 0;
+                padding: $whitespace;
+                transition: all $transition-duration $transition-timing-function;
+
+                &:hover {
+                  transform: translateY(- calc($whitespace / 2));
+                }
```

Comment:
```suggestion
```
I'm not sure I'm a fan of using this as the hover indication, it seems rather gimmicky. It also doesn't make much sense as it's a hover indication on a non-interactive element, which is confusing UX -- you can hover over the name of a member and get this visual feedback that the element is interactive, but clicking does nothing...

The only interactive element that seems to be missing a hover interaction is the member image itself -- if you wish to address that, let's address that on the interactive element itself (the image). I think a desaturation filter on hover would probably work well.

Reply:
I will try working on a desaturation filter! Thanks for the input, still figuring out how open source works lol, especially the what not to do part 😅

Reply:
I made the change as requested, do let me know if this works fine!